client/resolve.c

```diff
@@ -316,7 +316,7 @@ TDNFPrepareSinglePkg(
     dwError = SolvCountPkgByName(pSack, pszPkgName, &dwCount);
     if(dwError || dwCount == 0)
```

Comment:
ah. this was bad. thanks for finding it.
Can we please make it clear like
BAIL_ON_TDNF_ERROR(dwError);

if (dwCount == 0)
{
    dwError = ERROR_TDNF_NO_SEARCH_RESULTS;
    BAIL_ON_TDNF_ERROR(dwError);
}

Reply:
Sure, I'll do it.